test/views/IntViewSpec.scala

```diff
@@ -50,15 +50,15 @@ class IntViewSpec extends ViewSpec with Mappings {
 
     "IntView" - {
 
-      act.like(renderTitle(viewModelGen)(view(_), _.title.key))
-      act.like(renderHeading(viewModelGen)(view(_), _.heading))
-      act.like(renderInputWithLabel(singleIntViewModel)("value", view(_), _.heading))
-      act.like(renderInputWithLabel(tripleIntViewModel)("value.1", view(_), _.page.fields.head.label))
-      act.like(renderInputWithLabel(tripleIntViewModel)("value.2", view(_), _.page.fields(1).label))
-      act.like(renderInputWithLabel(tripleIntViewModel)("value.3", view(_), _.page.fields(2).label))
-      act.like(renderErrors(invalidViewModel)(view(_), _ => "int.error.required"))
-      act.like(renderForm(viewModelGen)(view(_), _.onSubmit))
-      act.like(renderButtonText(viewModelGen)(view(_), _.buttonText))
+      act.like(renderTitle(viewModelGen)(view(intForm, _), _.title.key))
+      act.like(renderHeading(viewModelGen)(view(intForm, _), _.heading))
+      act.like(renderInputWithLabel(singleIntViewModel)("value", view(intForm, _), _.heading))
+//      act.like(renderInputWithLabel(tripleIntViewModel)("value.1", view(tripleIntForm, _), _.page.fields.head.label))
```

Comment:
Is this commented out code the one that's going to be sorted out as a separate PR?

Reply:
Yes, these tests will be replaced by `MultipleQuestionViewSpec`, on a separate branch (e.g. `PSR-405-unit-tests`)